Content.Shared/_RMC14/Damage/SharedRMCDamageableSystem.cs

```diff
@@ -65,6 +66,9 @@ public abstract class SharedRMCDamageableSystem : EntitySystem
     private EntityQuery<VictimInfectedComponent> _victimInfectedQuery;
     private EntityQuery<XenoNestedComponent> _xenoNestedQuery;
 
+    private EntityQuery<RMCFireProjectileComponent> _fireProjectileQuery;
+
+    private EntityQuery<RMCFireProjectileImmuneComponent> _fireProjectileImmuneQuery;
```

Comment:
Same with these two queries, you also aren't initializing them so using them would crash the game
```suggestion
```